src/doc/cheatsheet.ts

```diff
@@ -47,7 +47,7 @@ export const doc_cheatsheet: Documentation = {
 	</table>
 	<table class="nicetable">
 	<tr><th><a href="?doc#/library">standard library</a></th><th>topic</th></tr>
-	<tr><td><a href="?doc#/library/core">code functions</a></td><td>messages, time, errors, files, events</td></tr>
+	<tr><td><a href="?doc#/library/core">core functions</a></td><td>messages, time, errors, files, events</td></tr>
```

Comment:
I'm honestly not a fan of unrelated drive-by changes, but since it doesn't seem to bother anybody else and is technically not wrong just keep it here 🤷🏼‍♂️